### PR TITLE
feat(docs): add coffee server documentation 

### PR DESCRIPTION
**Pull request by tareknaser360**

### CODE REVIEW

File: docs/docs-book/src/running-coffee-as-a-server.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+   'http://localhost:8080/remote/rm' \
+   -H 'accept: application/json' \
+   -H 'Content-Type: application/json' \
+   -d '{
+     "repository_name": "folgore"
+   }'
+ ```
+
+ ### List Remote Repositories
+
```
Comment (vincenzopalazzo) [inline docs/docs-book/src/running-coffee-as-a-server.md]: Remove all and keep just a single line `To see the available calls you can use the Swagger API available at http://localhost:8080`

**Review by vincenzopalazzo — CHANGES_REQUESTED** [review]
The docs can be simplified by just pointing the user to the swagger documentation. We do not need another doc file that do the same

In addition you can drop the commit b7c4728d845a2bcb368f3d9ee9c18d4218d070b7

File: docs/docs-book/src/running-coffee-as-a-server.md (hunk 1 above)
Comment (tareknaser360) [reply docs/docs-book/src/running-coffee-as-a-server.md]: What do you think of  0d9e720 ?

Comment (vincenzopalazzo) [reply docs/docs-book/src/running-coffee-as-a-server.md]: Much better but I leave some comments

**Comment by vincenzopalazzo** [comment]
This needs a rebase on master

**Review by vincenzopalazzo — APPROVED** [review]
ACK https://github.com/coffee-tools/coffee/pull/160/commits/57590b4ab6048fc7d6e5fd29dc34b8eb69ea2784

**Pull request merged**